wigwam/commands.py

```diff
@@ -218,6 +228,46 @@ def compile_cmake(tag: str, base: str, no_cache: bool = False) -> Image:
     )
 
 
+def cmake_install(tag: str, base: str, no_cache: bool = False) -> Image:
+    """
+    Produces an image with the compiled working directory code installed.
+
+    .. note:
+        With this image, the workdir is moved to $BUILD_PREFIX.
+
+    Parameters
+    ----------
+    tag : str
+        The image tag.
+    base : str
+        The base image tag.
+    no_cache : bool, optional
+        Run Docker build with no cache if True. Defaults to False.
+
+    Returns
+    -------
+    Image
+        The generated image.
+    """
+
+    prefixed_tag: str = prefix_image_tag(tag)
+    prefixed_base_tag: str = prefix_image_tag(base)
+
+    image: Image = Image(prefixed_base_tag)
+
+    is_64_bit = test_image(image=image, expression='"$BUILD_PREFIX/lib64"')
+
+    if is_64_bit:
+        lib = "lib64"
+    else:
+        lib = "lib"
```

Comment:
Let's add an assertion here to double-check that we're doing the right thing.

```suggestion
        lib = "lib"
        assert test_image(image=image, expression='"$BUILD_PREFIX/lib"')
```